solutions/usaco-965.mdx

```diff
@@ -88,4 +91,50 @@ int main() {
 		cout << cows[t] << '\n';
 	}
 }
-```
\ No newline at end of file
+```
+</CPPSection>
+
+<PySection>
+## Brute Force with Itertools
+
+<Warning>
+Due to the brute force nature of this solution, it is very slow and is not the optimal solution.
+</Warning>
+We can easily iterate through every possible permutation of cows in lexographic order using Itertools. Then we can check for the first permutation that meets the adjacency constraints and this will be our answer. 
+
+**Time Complexity:** $\mathcal{O}(NC \cdot C!)$, where $C$ represents the number of cows 
+```py
+from itertools import permutations
+
+NUM_COWS = 8
+
+data = open('lineup.in').read().strip().split('\n')
+n = int(data[0])
+
+cows = sorted(['Bessie', 'Buttercup', 'Belinda', 'Beatrice', 
```

Comment:
wait
instead of NUM_COWS why not just use len(cows) lol

Reply:
lol originally i just had 8 but i thought abt constants

Reply:
so you gonna change this or like

Reply:
ok sure